### ISSUE TITLE
ctrl+c doesn't successfully stop deployment

### ISSUE THREAD

**Issue by lovesegfault** [Infinisil/nixus#5]
When executing the deployment script `ctrl+c` is moot as it will just continue to go non-stop

**Comment by Infinisil** [comment]
Yeah this is rather annoying, I'm hoping that I can use Haskell instead of bash in the future, so I don't need to mess with bash's process handling (which is a nightmare imo)

**Comment by bqv** [comment]
I think a strength of this is that it doesn't require any extra binaries than nix and a shell, but I do like haskell so I won't complain
